db/seeds.rb

```diff
@@ -14,3 +14,21 @@
   ReasonType.create(description: criteria)
 end
 
+unless Rails.env.production?
+  agency = Agency.create
+  user = User.new
+  user.email = "admin@hifinf.org"
```

Comment:
true.  i just copied code from a migration that really just belongs in a seed.  Didn't give it more thought then that